pages/slack_guide.js

```diff
@@ -0,0 +1,31 @@
+import Head from 'components/head';
+import HeroBanner from 'components/HeroBanner/HeroBanner';
+import Content from 'components/Content/Content';
+import styles from 'styles/slack_guide.module.css';
+
+const pageTitle = 'Slack Guide';
+
+function SlackGuide() {
+  return (
+    <div className={styles.slackGuide}>
+      <Head title={pageTitle} />
+
+      <HeroBanner className={styles.hero} title={pageTitle} />
+
+      <Content
+        columns={[
+          <iframe
+            title="Slack Guide"
+            src="https://youtube.com/embed/m2JuAa6-ors"
+            frameBorder="0"
+            allowFullScreen
+            width="755"
+            height="425"
+          />,
+        ]}
+      />
```

Comment:
```suggestion
      <HeroBanner className={styles.hero} title={pageTitle}>
        <iframe
          title="Slack Guide"
          src="https://youtube.com/embed/m2JuAa6-ors"
          frameBorder="0"
          allowFullScreen
          width="755"
          height="425"
        />
      </HeroBanner>
```

try this for me, would you?